### PR TITLE
Feature/create mint nft #18

### PR DESCRIPTION
**Pull request by minhtuan412**
Add deloy NFTERC-721

### CODE REVIEW

File: api/src/twitter.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  new Web3.providers.HttpProvider(process.env.EVM_RPC_TARGET)
+);
+
+const EXPLORER_URL = process.env.EXPLORER_URL;
+
+var client = new Twitter({
+  consumer_key: "xGwoSRAdx8FQoc2WKYVGUsFtJ",
+  consumer_secret: "Ly0Bl8om7HGPToPjNoPp8KAlzhpmQiUTFJtmB9HW0tCAXWXhIv",
+  bearer_token:
+    "AAAAAAAAAAAAAAAAAAAAAD6zewEAAAAAY2m5dOB9nJGp3I%2FXqOZYvAqd%2Fpw%3DYwKOHI3nLvXxdxNtD0OZgw1bWGGanqvzgIw5pQ96iGOgylsh8z",
```
Comment (hanchenglin) [inline api/src/twitter.js:17]: please change to .env

File: api/src/twitter.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+});
+
+client
+  .post("update", { status: "I Love Twitter" })
```
Comment (hanchenglin) [inline api/src/twitter.js:21]: Please tell me twitter content!

File: api/package.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (hanchenglin) [inline api/package.json:7]: @minhtuan412  Please add script for start, we can start app using `yarn start`.

"start": "node src/index.js"

File: frontend/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+# build for production and launch server
+$ yarn build
+$ yarn start
+
+# generate static project
+$ yarn generate
+```
+
+For detailed explanation on how things work, check out the [documentation](https://nuxtjs.org).
```
Comment (hanchenglin) [inline frontend/README.md:20]: @minhtuan412  Please delete the unnecessary description below

File: frontend/components/dao/DaoPage.vue [hunk 5]
```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="dao">
+    <div class="md-layout dao-layout">
+      <div class="md-layout-item">
+        <div class="dao-item">
+            <span class="label">#天音かなた</span>
```
Comment (hanchenglin) [inline frontend/components/dao/DaoPage.vue:6]: @minhtuan412  This page is not a static page! Please get NTFs by your minted data. When you mint a NFT, should call a API to save to DynamoDB

File: frontend/components/home/HomePage.vue [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+import IMAGE from "@/assets/image/example.png";
+export default {
+  name: "HomePage",
+  components: {},
+  mixins: [],
+  props: {},
+  data() {
+    return {
+      IMAGE,
+      listImages: [
```
Comment (hanchenglin) [inline frontend/components/home/HomePage.vue:23]: @minhtuan412  This page is not a static page!
It is to get and display the post list with API. You should have talked to the issue and the meeting.

File: frontend/components/market-completed/MarketCompletedPage.vue [hunk 7]
```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="marketplace">
+    <div class="md-layout marketplace-layout">
+      <div class="md-layout-item">
+        <div class="marketplace-item">
+            <img :src="IMAGE" class="image"/>
+            <span class="text-detail">かなたそ</span>
```
Comment (hanchenglin) [inline frontend/components/market-completed/MarketCompletedPage.vue:7]: @minhtuan412  This page is not a static page!

File: frontend/components/marketplace/MarketPlacePage.vue [hunk 8]
```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="marketplace">
+    <div class="md-layout marketplace-layout">
+      <div class="md-layout-item">
+        <div class="marketplace-item">
+            <img :src="IMAGE" class="image"/>
```
Comment (hanchenglin) [inline frontend/components/marketplace/MarketPlacePage.vue:6]: @minhtuan412  This page is not a static page!

File: frontend/components/post/PostInputPage.vue [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+      <div class="md-layout-item">
+        <div class="post-box">
+            <span class="post-text">かなたそ</span>
+            <div class="image">
+                <img :src="IMAGE" />
+                <img :src="EXIT" class="exit"/>
+            </div>
+        </div>
+        <div class="info">
+            <img :src="POST" />
```
Comment (hanchenglin) [inline frontend/components/post/PostInputPage.vue:16]: @minhtuan412  Why can't I select an image?

File: mintNFT/artifacts/@openzeppelin/contracts/token/ERC721/IERC721.sol/IERC721.dbg.json [hunk 10]
```diff
@@ -0,0 +1,4 @@
+{
+  "_format": "hh-sol-dbg-1",
+  "buildInfo": "..\\..\\..\\..\\..\\build-info\\f549e9015178714682ce1a1a2f132cf7.json"
```
Comment (hanchenglin) [inline mintNFT/artifacts/@openzeppelin/contracts/token/ERC721/IERC721.sol/IERC721.dbg.json:3]: @minhtuan412  Please remove /artifacts  folder, do not need push to git!

**Comment by hanchenglin** [comment]
@minhtuan412  Please understand the screen contents properly.
ALL pages are dynamic page. Not static page! 
Do you still not understand the screen contents in the last two weeks?

File: api/src/twitter.js (hunk 1 above)
Comment (minhtuan412) [reply api/src/twitter.js]: I updated it

File: api/package.json (hunk 3 above)
Comment (minhtuan412) [reply api/package.json:8]: I updated it

File: frontend/components/dao/DaoPage.vue (hunk 5 above)
Comment (minhtuan412) [reply frontend/components/dao/DaoPage.vue:6]: I updated it

File: frontend/components/home/HomePage.vue (hunk 6 above)
Comment (minhtuan412) [reply frontend/components/home/HomePage.vue]: I updated it

File: frontend/components/market-completed/MarketCompletedPage.vue (hunk 7 above)
Comment (minhtuan412) [reply frontend/components/market-completed/MarketCompletedPage.vue]: I updated it

File: frontend/components/post/PostInputPage.vue (hunk 9 above)
Comment (minhtuan412) [reply frontend/components/post/PostInputPage.vue]: I updated it

File: mintNFT/artifacts/@openzeppelin/contracts/token/ERC721/IERC721.sol/IERC721.dbg.json (hunk 10 above)
Comment (minhtuan412) [reply mintNFT/artifacts/@openzeppelin/contracts/token/ERC721/IERC721.sol/IERC721.dbg.json]: I removed it

File: api/src/twitter.js (hunk 1 above)
Comment (hanchenglin) [reply api/src/twitter.js]: @minhtuan412 Now is  still hard-code

File: api/.env_sample [hunk 11]
```diff
@@ -0,0 +1,9 @@
+## dynamodb local for localhost
+USE_DYNAMODB_LOCAL=true
+DYNAMODB_LOCAL_URL=http://localhost:8000
+
+CONSUMER_KEY=xGwoSRAdx8FQoc2WKYVGUsFtJ
+CONSUMER_SECRET=Ly0Bl8om7HGPToPjNoPp8KAlzhpmQiUTFJtmB9HW0tCAXWXhIv
+BEARER_TOKEN=    "AAAAAAAAAAAAAAAAAAAAAD6zewEAAAAAY2m5dOB9nJGp3I%2FXqOZYvAqd%2Fpw%3DYwKOHI3nLvXxdxNtD0OZgw1bWGGanqvzgIw5pQ96iGOgylsh8z",
```
Comment (hanchenglin) [inline api/.env_sample:7]: BEARER_TOKEN=    "xxx"  has no problem?

File: frontend/nuxt.config.js [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+    { src: 'vue-material/dist/theme/default.css', lang: 'css' },
+  ],
+
+  // Plugins to run before rendering page: https://go.nuxtjs.dev/config-plugins
+  plugins: [
+    '@/plugins/vue-material',
+  ],
+
+  env: {
+    WEB3AUTH_CLIENT_ID: 'BH6RbucXSjsZ9wX9rQLwdilwmEtOYvkGYrb462Ok26ZDMWOnIDaM2NBHj3OdspWV9ZY0CEFKTyPGUwlYhB4c2pU', // get from https://dashboard.web3auth.io
```
Comment (hanchenglin) [inline frontend/nuxt.config.js:55]: do not use hardcode, please move to .env

Comment (minhtuan412) [reply frontend/nuxt.config.js:55]: I updated it

File: api/.env_sample (hunk 11 above)
Comment (minhtuan412) [reply api/.env_sample:7]: I need key twitter, so I have created twitter account to test and waiting to confirm
image

**Pull request merged**